docs/source/api/algorithms/clustering/kmeans-init.rst

```diff
@@ -86,6 +86,13 @@ Operation
       | :expr:`result.centroids.row_count == desc.cluster_count`
       | :expr:`result.centroids.column_count == input.data.column_count`
 
+
+-------------
+Usage example
```

Comment:
```suggestion
Example
```

Reply:
This is a title which is used in all other sections. Don't think it needs to be changed here